controller/router.js

```diff
@@ -95,6 +97,16 @@ module.exports = [
                         .number()
                         .integer()
                         .positive()
+                },
+                query: {
+                    fromDate: Joi
+                       .date()
+                       .format('YYYY-MM-DD')
+                       .default(moment().startOf('day').toDate()),
+                    toDate: Joi
+                       .date()
+                       .format('YYYY-MM-DD')
+                       .default((moment().add(addSevenDays, 'days')).toDate())
```

Comment:
To match the existing default,
I think going from 1 week ago to today would be best.
Which would look something like this:

``` js
fromDate: Joi
    .date()
    .format('YYYY-MM-DD')
    .default(
        moment()
        .startOf('day')
        .subtract(1, 'week')
        .toDate()
    ),
toDate: Joi
    .date()
    .format('YYYY-MM-DD')
    .default(
        moment()
        .startOf('day')
        .toDate()
    )
```